### ISSUE TITLE
how to identify CDR after numbering sequence

### ISSUE THREAD

**Issue by renyongzhe** [oxpig/ANARCI#39]
Hi ,all. 
Because [SCALOP](http://opig.stats.ox.ac.uk/webapps/newsabdab/sabpred/scalop) will miss H-CDR3, is there annother tool could identify CDR region? After numbering the antibody sequence, whether it's the right way to extract CDR region according to the position  of CDR [Definitions](http://bioinf.org.uk/abs/info.html#cdrdef) ? Any other suggestions would be appreciated.

**Comment by renyongzhe** [comment]
> 
Thanks for your interpretation. Annother format in IMGT numbered heavy chain sequence like below where CDR1 located in H26-H33. How to get the correct CDR1 sequence? The output of SCALOP is GFSINGSW.
[(26, S), (27, G), (28, F), (29, S),  (30, I), (31, -), (32, -), (33, -), (34, -), (35, N), (36, G), ...]

**Comment by gciaberi** [comment]
You should double check the CDR definitions table you are using because it seems wrong, in the [official IMGT website](https://www.imgt.org/IMGTScientificChart/Nomenclature/IMGT-FRCDRdefinition.html) the CDR1 is defined as H27-38.

**Comment by renyongzhe** [comment]
> You should double check the CDR definitions table you are using because it seems wrong, in the [official IMGT website](https://www.imgt.org/IMGTScientificChart/Nomenclature/IMGT-FRCDRdefinition.html) the CDR1 is defined as H27-38.

You are right. But that make me confused about this [definitions](http://bioinf.org.uk/abs/info.html#cdrdef) which many people and documents cite it.
